src/git-extract

```diff
@@ -54,17 +54,18 @@ function preconditions {
 }
 
 
-function steps {
-  echo_if_true "stash_open_changes" "$INITIAL_OPEN_CHANGES"
+function should_stash_open_changes {
```

Comment:
Good question. Technically I think it could live inside or outside of it. I don't think its really a pre-condition here. It could make more sense in the beginning of the steps function.